x-pack/plugin/rollup/src/main/java/org/elasticsearch/xpack/downsample/FieldValueFetcher.java

```diff
@@ -55,41 +56,39 @@ FormattedDocValues getLeaf(LeafReaderContext context) {
         return fieldData.load(context).getFormattedValues(format);
     }
 
-    Object format(Object value) {
```

Comment:
Removing this method because it was not use anywhere. Probably a left-over from old code